.gitignore

```diff
@@ -1,2 +1,5 @@
 .vscode
-.sfdx
\ No newline at end of file
+.sfdx
+force-app/main/default/aura/exampleComponent
```

Comment:
@nhelterbrand is there a reason to put this stuff in the gitignore?

Reply:
the .sfdx is the vscode salesforce extension folder that gets added automatically, and the exampleComponent was the component i was using to test those snippets in the README. are you saying that these should be included? i was trying to keep the repo clean and just have the lightning logger component stuff in there alone

Reply:
Sorry I meant these lines in the gitignore (not the .sfdx):
```force-app/main/default/aura/exampleComponent
force-app/main/default/flexipages
force-app/main/default/settings```

This would prevent us adding any flexipages or settings down the line as opposed to just not committing them to the repo.

Reply:
ahh ok, yeah for sure that makes sense